elasticjob-error-handler/elasticjob-error-handler-type/elasticjob-error-handler-dingtalk/src/main/java/org/apache/shardingsphere/elasticjob/error/handler/dingtalk/DingtalkConfiguration.java

```diff
@@ -33,4 +35,17 @@
     private final Integer connectTimeout;
     
     private final Integer readTimeout;
+    
+    /**
+     * Get dingtalk config.
+     *
+     * @param props props
+     * @return dingtalk config.
+     */
+    public static DingtalkConfiguration getByProps(final Properties props) {
```

Comment:
dingtalk.webhook，dingtalk.keyword and dingtalk.xxx.
It' better to use a constant field for this.